spec/clipboard.spec.js

```diff
@@ -142,5 +143,17 @@ describe('Clipboard', () => {
 
       expect(parseContent(div)[0]).toEqual('bar')
     })
+
+    it('changes absolute links to relative ones with the keepInternalRelativeLinks flag set to true', () => {
```

Comment:
Can you also add a test where nothing gets replaced?

```js
expect(extractSingleBlock('<a href="/test123">a</a>'))
  .toEqual('<a href="/test123">a</a>')
```

and maybe also:

```js
expect(extractSingleBlock('<a href="http://foo.com/test123">a</a>'))
  .toEqual('<a href="http://foo.com/test123">a</a>')
```

Reply:
Oh and:

```js
expect(extractSingleBlock('<a href="">')).toEqual('<a href="">')
```

and

```js
```js
expect(extractSingleBlock('<a href>')).toEqual('<a href>')
```
```

Reply:
Added and there were some previous tests for the last 2 cases the expected outcome is that it's removed
```js
expect(extractSingleBlock('<a href>')).toEqual('<a>')
```